pkg/render/dex_test.go

```diff
@@ -89,5 +105,16 @@ var _ = Describe("dex rendering tests", func() {
 				ExpectResource(resources[i], expectedRes.name, expectedRes.ns, expectedRes.group, expectedRes.version, expectedRes.kind)
 			}
 		})
+
+		It("should render the cluster dns properly", func() {
```

Comment:
This test is testing the dex key validator config is rendering properly

Reply:
It might be nice if we had a test for the non-default local search path case since we switch on that in the dex baseCfg

Reply:
There is currently no way to configure the resolv.conf path in `operator.tigera.io`. As discussed I moved a test to dns_test.go where this is tested instead.